### PR TITLE
Add all supported games to the quick switch menu

### PR DESCRIPTION
**Pull request by jeffvli**
## Preview
image

--- 
- Adds all supported games to the quick switch menu

I am not a C# developer so feel free to make any changes if necessary.

### CODE REVIEW

**Review by Jorixon — COMMENTED** [review]
Nice work! I was recently thinking about doing something like this :)

I have some minor code suggestions

File: src/GIMI-ModManager.WinUI/Views/ShellPage.xaml.cs [hunk 1]
```diff
@@ -159,61 +159,59 @@ private void GameServiceOnInitialized(object? sender, EventArgs e)
...
         App.MainWindow.DispatcherQueue.EnqueueAsync(async () =>
         {
-            var selectedGame = await ViewModel.SelectedGameService.GetSelectedGameAsync();
-            if (selectedGame == SupportedGames.WuWa.ToString() || selectedGame == SupportedGames.ZZZ.ToString())
-                return;
-
             var notSelectedGame = await ViewModel.SelectedGameService.GetNotSelectedGameAsync();
 
+            for (var i = notSelectedGame.Length - 1; i >= 0; i--)
+            {
```
[the remark this answers is not in the document]
Comment (jeffvli) [reply src/GIMI-ModManager.WinUI/Views/ShellPage.xaml.cs]: Switched to a foreach but used `notSelectedGame.Reverse()` to account for the ascending order returned from `GetNotSelectedGameAsync`.

File: src/GIMI-ModManager.WinUI/Views/ShellPage.xaml.cs [hunk 2]
```diff
@@ -345,9 +343,18 @@ private async void SwitchGameButtonOnDoubleTapped(object sender, DoubleTappedRou
...
+            var elementTag = element.Tag;
+
+            if (elementTag.GetType() == typeof(string))
+            {
+                var gameName = (string) elementTag;
+                await App.GetService<LifeCycleService>().RestartAsync(notifyOnError: true,
+                    postShutdownLogic: () => ViewModel.SelectedGameService.SetSelectedGame(gameName))
+                    .ConfigureAwait(false);
+            }
+        }
```
[the remark this answers is not in the document]
Comment (jeffvli) [reply src/GIMI-ModManager.WinUI/Views/ShellPage.xaml.cs]: Added!

**Comment by Jorixon** [comment]
If there is nothing else, you'd like to add, then I'll merge this in

**Review by Jorixon — APPROVED** [review]

**Comment by jeffvli** [comment]
Nothing else for this PR on my side.

**Pull request merged**